block/modes.go

```diff
@@ -87,6 +87,10 @@ func (m *Manager) runAsProposer(ctx context.Context, eg *errgroup.Group) error {
 	// Monitor and handling of the rotation
 	go m.MonitorProposerRotation(ctx)
 
+	uerrors.ErrGroupGoLog(eg, m.logger, func() error {
+		return m.MonitorBalances(ctx)
+	})
```

Comment:
IMO is better to have balances on both. even though full-nodes dont need to spend any balance, i guess is better to have balances monitored in both cases in case there are rotations...